tests/components/application-connector/Makefile

```
@@ -1,45 +1,47 @@
-NAMESPACE ?= "test"
-
-DOCKER_TAG ?= "PR-14743"
-DOCKER_PUSH_REPOSITORY ?= "eu.gcr.io/kyma-project"
+NAMESPACE = test
```

```suggestion
NAMESPACE ?= test
```
Why do we disallow changing namespace with an env variable?

---

I agree. Even though it sound as corner case it might be useful for local development. On prow this name will always be `test`.